allennlp/modules/transformer/transformer_pooler.py

```diff
@@ -0,0 +1,11 @@
+from allennlp.common import FromParams
+from allennlp.modules.transformer.activation_layer import ActivationLayer
+
+
+class TransformerPooler(ActivationLayer, FromParams):
```

Comment:
I think it would be good to document this and `ActivationLayer` to at least say what the input and output shapes look like and where this class is typically used.

Reply:
Yes, it's on my list to add documentation to all classes.